### ISSUE TITLE
BrowserWindow zoom factor jumps up and down when there's an iframe

### ISSUE THREAD

**Issue by fancyydk** [electron/electron#8620]
* Electron version: 1.4.15
* Operating system: Windows 10

### Expected behavior

When you open a BrowserWindow with a zoomFactor set to something other than 1.0,

for example,
window = new BrowserWindow({width: 800, height: 600, webPreferences: { zoomFactor: 1.5 }});

The zoom factor (or zoom level) should stay consistent and should not change. It gets more obvious if you have more stuff on the page.

### Actual behavior

If the page the window is loading has an iframe in it, then the zoom factor goes back and forth between 1.0 and and whatever you set (e.g. 1.5) until the iframe finishes loading.

### How to reproduce

**index.html**
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Hello World!</title>
  </head>
  <body>
    <h1>Hi!</h1>
    <iframe src="http://www.bing.com"></iframe>
  </body>
</html>
```

**main.js**
```
const electron = require('electron');
const url = require('url');
const path = require('path');
const app = electron.app;
const BrowserWindow = electron.BrowserWindow;
let mainWindow;
function createWindow () {
  mainWindow = new BrowserWindow({width: 800, height: 600, webPreferences: { zoomFactor: 1.5 }});

  mainWindow.loadURL(url.format({
    pathname: path.join(__dirname, 'index.html'),
    protocol: 'file:',
    slashes: true
  }));

  mainWindow.on('closed', function () {
    mainWindow = null;
  });
}

app.on('ready', createWindow);
app.on('window-all-closed', function () {
  if (process.platform !== 'darwin') {
    app.quit();
  }
});

app.on('activate', function () {
  if (mainWindow === null) {
    createWindow();
  }
});
```

**Comment by deepak1556** [comment]
Thanks for the test case, can confirm the behavior. https://github.com/electron/electron/pull/8537 will fix this.

**Comment by kevinsawicki** [comment]
Fixed by #8537
